src/server/config.rs

```diff
@@ -31,9 +29,6 @@ const DEFAULT_GRPC_CONCURRENT_STREAM: usize = 1024;
 const DEFAULT_GRPC_RAFT_CONN_NUM: usize = 10;
 const DEFAULT_GRPC_STREAM_INITIAL_WINDOW_SIZE: u64 = 2 * 1024 * 1024;
 const DEFAULT_MESSAGES_PER_TICK: usize = 4096;
```

Comment:
Can we move those configures to other place? Users will lost their changed config after upgrade TiKV.

Reply:
I think we need a feature: if there are some configure entries don't match respective struct, report error and exit. So users could know they should update their `tikv.toml` file.

Reply:
Assign it to me.

Reply:
Yes, and I think do it in this PR is better than later.